src/test/java/org/apache/accumulo/proxy/its/SimpleProxyBase.java

```diff
@@ -1463,34 +1455,15 @@ public void userManagement() throws Exception {
       client.changeLocalUserPassword(creds, user, password);
       assertTrue(client.authenticateUser(creds, user, s2pp(ByteBufferUtil.toString(password))));
     }
-
-    if (isKerberosEnabled()) {
```

Comment:
I'm not really sure we need any of these kerberos checks. If the AccumuloClient is correctly created from props, that client should transparently handle whether it uses a PasswordToken or a KerberosToken in the user's client properties file.

There are a lot of places in the proxy where we have special handling for kerberos... especially for the tests. I'm not actually sure if the proxy supports, or ever supported, running as a kerberos-enabled service itself. If that's something we want to support, it will probably require substantial follow-on work. For now, it might just be useful to know whether the old proxy did or did not support that.